libs/paths/Realpath.ml

```diff
@@ -38,27 +38,14 @@
 (* Types *)
 (*****************************************************************************)
 
-(* TODO? use of Fpath.t instead? or too confusing? *)
```

Comment:
please do not rename to Realpath.ml. I used Rpath on purpose to match Fpath, Ppath.